### DISCUSSION TITLE
参考文献作为小标题在chapter下？

### DISCUSSION THREAD

**Discussion by PinkR1ver** [TheNetAdmin/zjuthesis#244] [General]
![image](https://user-images.githubusercontent.com/50662650/167242281-14cd9de3-8f4f-4518-9a79-2378578407e9.png)
overleaf的模板上参考文献在chapter下级标题下，怎么改成独立的呢？

**Comment by TheNetAdmin** [comment]
你是想去掉“参考文献”前置的编号？还是说要把“参考文献”提高到chapter层级？

**Reply by TheNetAdmin** [reply]
参考文献标题设定在这里

https://github.com/TheNetAdmin/zjuthesis/blob/f74b9d27bc3c757355ac0b7238fe85341f490dfe/config/format/general/heading.tex#L112-L123

**Reply by PinkR1ver** [reply]
感谢，是想把参考文献提到chapter层级，不过本科毕业论文中似乎是不需要chapter层级，现在去掉了chapter全部使用section来做了。谢谢！💖
